kernelci/build.py

```diff
@@ -1385,6 +1386,60 @@ def install(self, verbose=False):
         return super().install(verbose)
 
 
+class MakePerfTool(Step):
```

Comment:
How about `MakePerf`, and have a `kci_build make_perf` command?  That's what I typed initially and was confused why it didn't work.  Does anyone usually refer to this as `perftool` rather than `perf`?